src/Ipn/Handler.php

```diff
@@ -124,6 +128,9 @@ public function getEntity(array $ipn_data) {
    * @param array $ipn_data
    *   The data of the IPN request.
    *
+   * @return mixed
+   *   Returns the group transaction ID if the IPN type is supported.
+   *
```

Comment:
**CHANGE REQUEST**

`Returns the IPN group if the IPN is of a type that is supported by this module.`